pkg/storaged/client.js

```diff
@@ -106,37 +120,40 @@ define([
          * efficient since it reduces the number of D-Bus calls done
          * by the cache.
          */
-        return client.storaged_client.proxies(iface, "/org/storaged/Storaged",
+        return client.storaged_client.proxies(STORAGED_IFACE_PFX + "." + iface,
```

Comment:
I think the context is enough.  The whole udisks job complication can be removed once storaged takes over the role of udisks2.  At that point I think we should consider removing all traces of "storaged" from our code.